src/js/player.js

```diff
@@ -2232,6 +2232,11 @@ class Player extends Component {
         callback(this.play());
       };
 
+      // if we are in Safari, there is a high chance that loadstart will trigger after the gesture timeperiod
+      // in that case, we need to prime the video element by calling load so it'll be ready in time
+      if (browser.IS_ANY_SAFARI) {
```

Comment:
this is only done in safari to minimize any potential impact from calling load, can be loosed in the future, should we need it.

Reply:
Is this a concern for iOS Web Views?

Reply:
If it happens on desktop safari, I can't imagine why it can't happen on iOS as well. We'd need to test it, but theoretically, the extra call to load shouldn't do anything bad.

Reply:
I mean that should this block also apply to web views, specifically that `IS_ANY_SAFARI` excludes mobile Chrome

Reply:
That's a good question. Let me see how iOS Chrome handles autoplay restrictions.

Reply:
Looks like chrome iOS has the same issue. Let me check Android as well.

Reply:
Not seeing the behavior on Android. So, adding IS_IOS as well.